### PR TITLE
Todo project

### PR DESCRIPTION
**Pull request by des-des**
Okay so I have gone through, done some bug fixing and made the readme better!

Any feedback appreciated!

related #184

### CODE REVIEW

File: coursebook/fragments/projects/tdd-todo/README.md [hunk 1]
```diff
@@ -14,12 +14,109 @@ Some things you might want to add for the third part
   * style using css
   * remove the mark button and listen for clicks on the whole todo item.
 
+### setup
+
 Todo:
   * read through the file todo.js. Do not worry if it does not all makes sense at first.
   * set up a new repo for your project this week.
   * copy and commit the files inside skeleton folder into your repository. **warning** only one person in you time should do this, then everyone else can pull down
```
Comment (mantagen) [inline coursebook/fragments/projects/tdd-todo/README.md]: in *your team*

File: coursebook/fragments/projects/tdd-todo/README.md [hunk 2]
```diff
@@ -14,12 +14,109 @@ Some things you might want to add for the third part
...
+var todos = [
+  {id: 0, description: 'make tea', done: false},
+  {id: 1, description: 'make eggs', done: true},
+]
+var teaMade = todo.todoFunctions.markTodo(todos, 0);
+// [
+//   {id: 0, description: 'make tea', done: true},
+//   {id: 1, description: 'make eggs', done: true},
+// ]
+var coffeeNotMade = todo.todoFunctions.markTodo(todos, 1);
```
Comment (mantagen) [inline coursebook/fragments/projects/tdd-todo/README.md]: i feel like assigning to variables is a bit confusing here and not necessary

**Comment by JMurphyWeb** [comment]
@des-des looks good. After seeing it in use with FACN1, I really think these concepts should be taught during the week leading up to this project so the students do not feel overwhelmed by all the boiler-plate.

**Comment by des-des** [comment]
@nelsonic Yeah, tested it on humans, this pull is post test.. I am sure there is more to improve.

**Comment by des-des** [comment]
Merging as brad is on holiday

**Pull request merged**